docs/pages/develop/development-builds/share-with-your-team.mdx

```diff
@@ -31,6 +32,15 @@ You can also direct your teammate to the build page in the Expo dashboard. From
   src="/static/images/share-dev-build/expo-dashboard.png"
 />
 
+### Get development build with EAS CLI
```

Comment:
```suggestion
### Use EAS CLI
```

Since this whole document talks about development build. We can just say that this section is about using EAS CLI.